synchronization/src/main/java/de/cyface/synchronization/HttpConnection.java

```diff
@@ -411,14 +432,22 @@ String generateHeader(@NonNull final SyncAdapter.MetaData metaData, @NonNull fin
         // To support the API v2 specification we may not change the "vehicle" key name of the modality
         final String modalityPart = generatePart("vehicle", String.valueOf(metaData.modality.getDatabaseIdentifier()));
 
-        // File meta data
```

Comment:
The file part is now written together with the file content not in here in the header (which would not work with two file parts as the file content of the first file needs to come before the second file part header.